server/py/services/api/crud/tags.py

```diff
@@ -46,7 +46,11 @@ def overwrite_object_tags_with_tag(
         )
         if not overwrite_func:
```

Comment:
id validate the tag on the endpoint before calling the CRUD to avoid thread allocation - wdyt?